### PR TITLE
singleton

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: examples/singleton/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Auth filter example
+
+You can run the example like so:
+
+```
+npm run asbuild
+podman run -ti --rm -p 8080:8080 -p 8001:8001 --entrypoint=envoy -v $PWD/envoycfg.yaml:$PWD/envoycfg.yaml:ro -v $PWD/build:$PWD/build:ro -w $PWD docker.io/envoyproxy/envoy-dev:f6679d51cc7b2f0b5e05c883a035ad87d011f454 -c $PWD/envoycfg.yaml
```
Comment (Sodman) [inline examples/singleton/README.md:7]: Do we need this particular sha of `envoy-dev`? Will this work with `v1.17.0`? Would be nice to specify a version for context if anyone's looking at this in the future after a few more Envoy releases.

File: examples/singleton/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --use abort=abort_proc_exit --sourceMap --optimize",
+    "asbuild": "npm run asbuild:untouched && npm run asbuild:optimized",
+    "test": "node tests"
+  },
+  "dependencies": {
+    "assemblyscript": "^0.14.8",
+    "@solo-io/proxy-runtime": "file:../.."
+  },
+  "devDependencies": {
+    "assemblyscript": "^0.14.8"
```
Comment (Sodman) [inline examples/singleton/package.json:13]: Do we need this as a devDependency, when it's already in dependencies? (or vice-versa?)

**Review by Sodman — COMMENTED** [review]
Looks Good to me, two minor question about enovy versions & package.json dependencies!

File: examples/singleton/README.md (hunk 1 above)
Comment (yuval-k) [reply examples/singleton/README.md:7]: it won't unfortunately; my patch was merged after this release

File: examples/singleton/package.json (hunk 2 above)
Comment (yuval-k) [reply examples/singleton/package.json:13]: i think we only need devDependency; but i'm not sure.. that's why i just updated the version numbers so they all match

**Review by Sodman — APPROVED** [review]

**Pull request merged**